remote-build.exp

```diff
@@ -123,6 +128,13 @@ if { ! [file exists configs/${at_version}] } {
 puts "AT version: $at_version"
 puts "Servers: $servers"
 
+# Define default value of variables if not previously defined
+# Directory used by the script to save its files.
+set_def tray_dir "~/$basedir/tmp/at-build-tray"
```

Comment:
IMHO we don't need to use the tmp dir, we can use ~/$basedir/at-build-tray directly.
This is just an opinion and should not block the patch.